### PR TITLE
style(board): pulsating real-time icon

### PR DESCRIPTION
**Pull request by purusott**
Tested on older browsers for compatilibity and our designer approved it visually
Please test this locally as well 🤝 
I have noticed the pulsating icon appears to be different in size on the landing page and preview (in /edit) due to screensizes, let me know if that`s an issue.

Thanks

Before
-
https://github.com/user-attachments/assets/953e2087-b987-4e2f-838f-a6d6b3caf459

After
- 

https://github.com/user-attachments/assets/cd0cb34f-de57-4aa3-9423-d47fbed710e6

### CODE REVIEW

File: tavla/app/globals.css [hunk 1]
```diff
@@ -241,7 +241,7 @@
     }
 
     .pulse {
-        @apply inline-flex rounded-full bg-success h-em-0.75 w-em-0.75 left-[calc(50%-(0.75em/2))] bottom-[calc(50%-(0.75em/2))];
+        @apply inline-flex rounded-full bg-success h-3 w-3 left-[calc(50%-((0.75em)/2))] bottom-[calc(50%-((h-3)/2))];
```
Comment (SelmaBergstrand) [inline tavla/app/globals.css:244]: bør `left-[calc(50%-((0.75em)/2))]` endres fra 0.75em til w-3? Så fjernes em-enheten helt

**Review by SelmaBergstrand — COMMENTED** [review]
funker fint for meg lokalt!

File: tavla/app/globals.css (hunk 1 above)
Comment (emilielr) [reply tavla/app/globals.css:244]: Hvis det går, så burde vi fjerne ja ☝️

Comment (emilielr) [reply tavla/app/globals.css:244]: Tenker vi bare har det slik det er med em :D

**Review by emilielr — APPROVED** [review]

**Pull request merged**